docs/apm/troubleshooting.asciidoc

```diff
@@ -95,13 +95,19 @@ Instead, we should strip away the unique information and group our transactions
 In this case, that means naming all blog transactions, `/blog`, and all documentation transactions, `/guide`.
 
 If you feel like you'd be losing valuable information by following this naming convention, don't fret!
-You can always add additional metadata to your transactions using {apm-guide-ref-v}/metadata.html#labels-fields[labels] (indexed) or
-{apm-guide-ref-v}/metadata.html#custom-fields[custom context] (non-indexed).
+You can always add additional metadata to your transactions using {apm-guide-ref}/metadata.html#labels-fields[labels] (indexed) or
+{apm-guide-ref}/metadata.html#custom-fields[custom context] (non-indexed).
 
 After ensuring you've correctly named your transactions,
-you might still see an error in the APM app related to too many transaction names.
-If this is the case, you can increase the default number of transaction groups displayed in the APM app by configuring
-<<apm-settings-kb,`xpack.apm.ui.transactionGroupBucketSize`>>.
+you might still see errors in the APM app related to transaction group limit reached:
+
+`The transaction group limit in APM Server has been reached. Excess groups have been captured in **_other**. Please see the {apm-guide-ref}/transaction-metrics.html#transactions-max_groups[APM Server docs] for **aggregation.transaction.max_groups** to increase this limit`
```

Comment:
We should also explain how to fix this in the APM integration and ESS:

ESS: `apm-server.aggregation.transactions.*`  https://www.elastic.co/guide/en/cloud/current/ec-manage-apm-settings.html
APM integration: ?